apps/src/scratch/scratch.js

```diff
@@ -14,7 +14,7 @@ import { singleton as studioApp } from '@cdo/apps/StudioApp';
 import ScratchView from './ScratchView';
 import { scratchDefaultProject } from './scratchDefaultProject';
 
-const Scratch = window.Scratch = window.Scratch || {};
+const Scratch = window.__TestInterface__ = {};
```

Comment:
Hmm, I was only able to find one match and it seems like it shouldn't conflict: https://github.com/code-dot-org/code-dot-org/search?q=__TestInterface__

Am I missing some other usages?